tools/ktx/command_create.cpp

```diff
@@ -846,7 +848,7 @@ class CommandCreate : public Command {
 
 private:
     void executeCreate();
-    void encode(KTXTexture2& texture, OptionsCodec<false>& opts);
+    void encode(KTXTexture2& texture, OptionsBasis<false>& opts);
```

Comment:
Perhaps we should call this `encodeBasis`.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/817/commits/dd12a5ec82f53758c77cbf18c7c2f6e2443a30ff